tests/driver_netdev_common/Makefile

```
@@ -6,8 +6,8 @@ USEMODULE += shell_commands
 USEMODULE += ps
 
 # allow to use a smaller packet buffer in low-PDU drivers
-ifdef GNRC_PKTBUF_SIZE
-  CFLAGS += -DGNRC_PKTBUF_SIZE=$(GNRC_PKTBUF_SIZE)
+ifdef CONFIG_GNRC_PKTBUF_SIZE
+  CFLAGS += -DCONFIG_GNRC_PKTBUF_SIZE=$(CONFIG_GNRC_PKTBUF_SIZE)
 endif
```

This will clash with the value provided by Kconfig. I think we should keep the makefile variable name
```suggestion
ifdef GNRC_PKTBUF_SIZE
  CFLAGS += -DCONFIG_GNRC_PKTBUF_SIZE=$(GNRC_PKTBUF_SIZE)
endif
```
Also, needs to be changed back on other makefiles that include this (like `tests/driver_xbee/Makefile`).

---

Yes. thanks for pointing that out.

---

There are multiple make files that make use of the variable. The variable is not well documented which you did mention. I hope this won't create confusion among users.